test/runtime/specs/juttle-spec/stdlib/select.spec.md

```diff
@@ -179,3 +179,63 @@
     {parity:1, x: 15}
     {parity:0, x: 36}
     {parity:1, x: 35}
+
+## top selects the 5 largest values
+### Juttle
+    import "select.juttle" as select;
+    emit -limit 10 -from Date.new(0)
+    | put x = 10-count()
+    | select.top -n 5 -by 'x'
+    | keep x
+    | view result
+
+### Output
+    {x: 5}
+    {x: 6}
+    {x: 7}
+    {x: 8}
+    {x: 9}
```

Comment:
Interesting, I’d expect the sequence to be reversed. My mental model for `top` is “sort so that the biggest value is first and then cut-off *n* items”. Think hi-score in games, various [rating tables](http://www.goratings.org/), etc.